person.rb

```diff
@@ -0,0 +1,37 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..99_999)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def can_use_services?
```

Comment:
- [ ] [OPTIONAL] here we can just call `of_age?() || @parent_permission` without explicitly indicating the return type.